ipaclient/remote_plugins/__init__.py

```diff
@@ -41,8 +41,13 @@ def _read(self):
         try:
             with open(self._path, 'r') as sc:
                 self._dict = json.load(sc)
-        except EnvironmentError as e:
-            if e.errno != errno.ENOENT:
+        except (EnvironmentError, ValueError) as e:
```

Comment:
I guess you should 'except Exception as e:' here, same as in https://github.com/freeipa/freeipa/blob/master/ipaclient/remote_plugins/schema.py#L387